packages/agents/lighthouse/src/tasks/prover/operations/publisher.ts

```diff
@@ -276,6 +275,7 @@ export const enqueue = async () => {
                   throw new NoMessageRootCount(originDomain, targetMessageRoot);
                 }
                 const batchSize = config.proverBatchSize[destinationDomain] ?? DEFAULT_PROVER_BATCH_SIZE;
```

Comment:
This is currently both, minimum and maximum for a batch size.

What are your thoughts on having a range ?
`batchSize` is min, and adding a new `maxBatchSize` as max,
This allows us to, push as many messages as we can, without much risk

If so, what can the max be ?

Reply:
I don't see the clear improvement than having only `batchSize` but it requires us to add more config.